packages/shared/src/components/squads/moderation/SquadModerationItem.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { ReactElement } from 'react';
+import {
+  Typography,
+  TypographyTag,
+  TypographyType,
+} from '../../typography/Typography';
+import { CardImage } from '../../cards/common/Card';
+import PostTags from '../../cards/common/PostTags';
+import { SourcePostModeration } from '../../../graphql/squads';
+import { SquadModerationActions } from './SquadModerationActions';
+import { ProfileImageSize, ProfilePicture } from '../../ProfilePicture';
+import PostMetadata from '../../cards/common/PostMetadata';
+
+interface SquadModerationListProps {
+  data: SourcePostModeration;
+  onApprove: (id: string) => void;
+  onReject: (id: string) => void;
+  isLoading: boolean;
+}
+
+export function SquadModerationItem({
+  data,
+  onReject,
+  onApprove,
+  isLoading,
+}: SquadModerationListProps): ReactElement {
+  const { post } = data;
+
+  return (
+    <div className="flex flex-col gap-4 p-6">
```

Comment:
The intricacies of which status to display based on the user's role is going to be raised in a separate PR. From this ticket: https://dailydotdev.atlassian.net/jira/software/projects/MI/boards/31?selectedIssue=MI-573